plugins/inputs/cisco_telemetry_mdt/cisco_telemetry_mdt.go

```diff
@@ -50,13 +52,48 @@ type CiscoTelemetryMDT struct {
 
 	// Internal state
 	aliases   map[string]string
+	dmes      map[string]string
```

Comment:
Can you please rename this (e.g. to `dmesFuncs` or similar) or `Dmes` above to avoid confusion between the two.